docs/Rakefile

```diff
@@ -6,7 +6,12 @@ task :default => :gen
 BUILD_DIR          = "build"
 MARKDOWN_FILES     = Rake::FileList.new("source/_tmp/*.md")
 TOPIC_MAP          = "#{BUILD_DIR}/_topic_map.yml"
-ADOC_VARIABLES     = "source/variables.adoc"
+ADOC_VARIABLES     = FileList.new('source/variables.adoc',
+ 'source/contributing/variables.adoc',
+ 'source/getting-started/variables.adoc',
+ 'source/openshift/variables.adoc',
+ 'source/using/variables.adoc'
+)
```

Comment:
> You broke it down because you thought there would be name/vale clash for variables?

To be clear, the files are identical and need to be. This is not about potentially having different content in each of them. This is about making sure that variable interpolation works for us using Middleman as well as integration into docs.openshift.org.